core/client/fs/src/main/java/alluxio/client/metrics/ClientMasterSync.java

```diff
@@ -33,41 +37,55 @@
  * it before retrying.
  */
 @ThreadSafe
-public final class ClientMasterSync implements HeartbeatExecutor {
-  private static final Logger LOG = LoggerFactory.getLogger(ClientMasterSync.class);
+public final class ClientMasterSync {
+  private static final Logger SAMPLING_LOG =
+      new SamplingLogger(LoggerFactory.getLogger(ClientMasterSync.class), 30 * Constants.SECOND_MS);
 
-  /** Client for communicating to metrics master. */
+  /**
+   * Client for communicating to metrics master.
+   */
```

Comment:
Could you change this to `/** Comment */` format?

Reply:
done